naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/controllers/GraphQLController.scala

```diff
@@ -147,7 +147,11 @@ class GraphQLController @Inject()(
               }
           }.recover {
             case error: QueryAnalysisError =>
-              OutgoingQuery(error.resolveError.as[JsObject], None)
+              OutgoingQuery(
+                Json.obj(
```

Comment:
Do we need to `logger.warn` or `logger.error` here? is this frequent or useful?

Reply:
This is a validation error, should we still log this?

Reply:
Is there still a means to get the error?  Usually this is handled by logging the full error with unique token and then returning just that token through the API.

Reply:
This is just a temporary solution that we wanted to release before the code freeze tomorrow. We will work on a proper fix later.

Reply:
It doesn't have to be fancy.  Create a random UUID, log it and the error, then return the UUID.  I don't think it's a good idea to obfuscate diagnostics right before the holidays.

Reply:
+1 for `errorCode`, and I feel like `message omitted for security` is undesired.

Reply:
There are other error handlers returning text: 'case error: Exception', 'case error: ErrorWithResolver ', and GraphQLController.exceptionHandler(...).  That last one has a TODO that looks security related.

Reply:
Considering that this has been long-existing behavior, it's more fine to keep it through the holidays than to jam in a potentially-disruptive change just because of the code freeze.

Reply:
I am working on generating random UUID and logging it as @kmcmurtrieca suggested